### DISCUSSION TITLE
GitHub Pages not showing all content

### DISCUSSION THREAD

**Discussion by shailesh-vaidya** [community/community#23499] [Other Feature Feedback, Questions, & Ideas]
<p>Hello Team,</p>
<p>I am using GitHub Pages to display the Doxygen report. I followed the below steps,</p>
<ul>
<li>Created branch named doxygen-automation</li>
<li>Created Doxygen report.</li>
<li>Renamed html folder as docs</li>
<li>Configured GitHub Pages from doxygen-automation branch and docs folder</li>
</ul>
<p>Github Pages is working as expected however few pages are missing.</p>
<p>Repo Used - <a href="https://github.com/shailesh-vaidya/cortx-motr" class="inline-onebox" rel="noopener nofollow ugc">GitHub - shailesh-vaidya/cortx-motr: CORTX Motr is a distributed object and key-value storage system targeting mass capacity storage configurations. It's the core component of CORTX storage system.</a><br>
GitHub Wepage - <a href="https://shailesh-vaidya.github.io/cortx-motr/" class="inline-onebox" rel="noopener nofollow ugc">Motr: Main Page</a></p>
<p>Link not working from GitHub Webpage - <a href="https://shailesh-vaidya.github.io/cortx-motr/d7/d16/_recovery.html" rel="noopener nofollow ugc">https://shailesh-vaidya.github.io/cortx-motr/d7/d16/_recovery.html</a><br>
File is available in repo - <a href="https://github.com/shailesh-vaidya/cortx-motr/blob/13791360b9dd65610430a483a3618f4b9bc15531/docs/d7/d16/_recovery.html" class="inline-onebox" rel="noopener nofollow ugc">cortx-motr/_recovery.html at 13791360b9dd65610430a483a3618f4b9bc15531 · shailesh-vaidya/cortx-motr · GitHub</a></p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Do you have a <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#static-site-generators"><code>.nojekyll</code> file</a> in your <code>docs/</code> directory? By default the Pages site gets run through Jekyll, and Jekyll <a href="https://jekyllrb.com/docs/structure/">omits files and directories starting with <code>_</code></a> (among others).</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="shailesh-vaidya" data-post="1" data-topic="245706">
<div class="title">
<div class="quote-controls"></div>
 shailesh-vaidya:</div>
<blockquote>
<p>File is available in repo - <a href="https://github.com/shailesh-vaidya/cortx-motr/blob/13791360b9dd65610430a483a3618f4b9bc15531/docs/d7/d16/_recovery.html">cortx-motr/_recovery.html at 13791360b9dd65610430a483a3618f4b9bc15531 · shailesh-vaidya/cortx-motr · GitHub</a></p>
</blockquote>
</aside>
<p>it is not on main branch, please take note of that, because if that is really the case, that will be 404</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/f/7fa09d797d3f02899e39aaf606d436e30aa0f049.png" data-download-href="/uploads/short-url/id2KOyjZixiW6PObFBaX7srQ11T.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077434-ca52aec9-9a76-41f2-a12c-f882709c8474.png" alt="image" data-base62-sha1="id2KOyjZixiW6PObFBaX7srQ11T" width="379" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/f/7fa09d797d3f02899e39aaf606d436e30aa0f049_2_379x500.png, https://aws1.discourse-cdn.com/github/original/3X/7/f/7fa09d797d3f02899e39aaf606d436e30aa0f049.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/f/7fa09d797d3f02899e39aaf606d436e30aa0f049.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/f/7fa09d797d3f02899e39aaf606d436e30aa0f049_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">520×685 36.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>I don’t know the entire project, but that’s what I’m thinking of,</p>

**Comment by mishalora** [comment]
<p>This repos working epo Used - <a href="https://shoespundit.com/best-slip-on-work-boots-for-concrete" rel="noopener nofollow ugc">GitHub - shailesh-vaidya/cortx-motr: CORTX Motr is a distributed object and key-value storage system targeting mass capacity storage configurations. It’s the core component of CORTX storage system. </a></p>

**Comment by shailesh-vaidya** [comment]
<p><a class="mention" href="/airtower-luna">@airtower-luna</a> Thanks for the suggestion. This worked for me.</p>
